submissions/korotushak96/memory-pair-game/js/script.js

```diff
@@ -88,6 +93,8 @@ const gameInit = ()=>{
     scene.addEventListener('click', (event)=>{
         let {target} = event;
         target = target.parentNode.parentNode;
+        if(target.tagName != 'DIV') return;
+        console.log(target);
```

Comment:
As I know, PR should not contain any `console.log`. I recommend you to check [this article](https://javascript.info/debugging-chrome) about debugging, it might save you some time in the future.

Reply:
thanks. I'll remove it in next commit